packages/ui/src/message-room-details-dialog/components/PrivateChannel.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ChannelModal from "./ChannelModal";
+import styles from "./archive-channel.module.css";
+
+const PrivateChannel = ({ closeEdit }) => {
+  const org_id = localStorage.getItem("currentWorkspace");
+  const user = JSON.parse(sessionStorage.getItem("organisations"));
```

Comment:
Why are you getting the users from a session storage item called organisations. There's user data stored in local storage